.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+## Overview
+[link to Github Issue]
```

Comment:
This is kind of a small thing but I've found when working with templates that it's annoying to have to delete text. Maybe there's a way to write this so you can just add text instead of having to delete, e.g. make this line:
```
Task: 
```
And then you can just fill in the Github Issue

Reply:
Since Markdown supports HTML you can add HTML comments like `<!-- add a link to a Github Issue here -->`.